stream-chat-android-state/src/main/java/io/getstream/chat/android/offline/event/handler/internal/EventHandlerImpl.kt

```diff
@@ -193,7 +193,7 @@ internal class EventHandlerImpl(
             }
     }
 
-    private suspend fun replayEventsForAllChannels(user: User) {
+    private suspend fun replayEventsForAllChannels() {
```

Comment:
Same here (revert). I would appreciate it

Reply:
Reverted :smile: